src/styles/styles.js

```diff
@@ -1691,14 +1691,14 @@ const styles = {
         backgroundColor: themeColors.modalBackground,
     },
     PDFView: {
-        flex: 1,
+        display: 'grid',
```

Comment:
I think we should add a comment here to avoid setting a bad example. Something like:

```
`display: grid` is not supported in native platforms! It's being used on Web/Desktop only to vertically center PDFs without it overflowing the top of the file, which happens when using `display: flex`. 
```